example/types/ComponentLabels.ts

```diff
@@ -1,3 +1,8 @@
+const appNavigationLabels = {
+  USE_ROOM_MANAGER_TAB: 'use_room_manager_tab',
+  USE_TOKEN_TAB: 'use_token_tab',
```

Comment:
```suggestion
ROOM_MANAGER_TAB: 'room_manager_tab',
TOKEN_TAB: 'token_tab',
```
This sounds more natural.